src/shared/crypto/bitcoin/bip39/bip39.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { pbkdf2 } from '@noble/hashes/pbkdf2';
+import { sha512 } from '@noble/hashes/sha512';
+
+// Code ported from https://github.com/bitcoinjs/bip39/
+function normalize(str?: string): string {
+  return (str || '').normalize('NFKD');
+}
+function salt(password?: string): string {
+  return 'mnemonic' + (password || '');
+}
+export function mnemonicToSeedSync(mnemonic: string, password?: string): Buffer {
+  const mnemonicBuffer = Uint8Array.from(Buffer.from(normalize(mnemonic), 'utf8'));
+  const saltBuffer = Uint8Array.from(Buffer.from(salt(normalize(password)), 'utf8'));
+  const res = pbkdf2(sha512, mnemonicBuffer, saltBuffer, {
+    c: 2048,
+    dkLen: 64,
+  });
+  return Buffer.from(res);
+}
```

Comment:
We don't need this, `@scure/bip39` has the same function https://github.com/paulmillr/scure-bip39#:~:text=function%20mnemonicToSeedSync(mnemonic%3A%20string%2C%20passphrase%3F%3A%20string)%3A%20Uint8Array%3B